src/main/java/httpserver/httpmessages/HTTPResponse.java

```diff
@@ -0,0 +1,42 @@
+package httpserver.httpmessages;
+
+public class HTTPResponse {
+
+    private final String reasonPhrase;
+    private final String statusCode;
```

Comment:
Is there a reason the `statusCode` is not an integer? I am 97.347% confident there are only numeric values for status codes